### PR TITLE
Add parameters: numberOfRecentPosts and numberOfFeaturedPosts 

### PR DESCRIPTION
**Pull request by razonyang**

### CODE REVIEW

File: layouts/partials/sidebar.html [hunk 1]
```diff
@@ -5,15 +5,18 @@ <h2>{{ .Site.Params.Author }}</h2>
       {{ .Site.Params.IntroDescription }}
     </div>
     <a href='{{ absLangURL "about/" }}' class="button mt-1" role="button">{{ i18n "read_more" }}</a>
+    {{- $numberOfFeaturedPosts := default 8 .Site.Params.numberOfFeaturedPosts }}
+    {{- $featuredPosts := first $numberOfFeaturedPosts (where (where .Site.RegularPages "Params.featured" true) "File.UniqueID" "!=" .File.UniqueID) }}
+    {{- if $featuredPosts }}
```
Comment (onweru) [inline layouts/partials/sidebar.html:10]: use with instead of if, it's more concise

File: layouts/partials/sidebar.html [hunk 2]
```diff
@@ -8,13 +8,12 @@ <h2>{{ .Site.Params.Author }}</h2>
     <a href='{{ absLangURL "about/" }}' class="button mt-1" role="button">{{ i18n "read_more" }}</a>
     <h2 class="mt-4">{{ i18n "featured_posts" }}</h2>
     <ul>
-    {{ range where .Site.RegularPages "Params.featured" true }}
-     {{- if ne $title .Title }}
```
Comment (onweru) [inline layouts/partials/sidebar.html:12]: need this check `{{- if ne $title .Title }}`, see conversation in #19

Comment (razonyang) [reply layouts/partials/sidebar.html:12]: Current viewing post is filtered by `"File.UniqueID" "!=" .File.UniqueID` already.

**Review by onweru — APPROVED** [review]
thanks

**Pull request merged**